frontend/src/views/SyncDatabaseSchema/index.vue

```diff
@@ -426,6 +427,7 @@ watch(
       const changeHistoryList = (
         await changeHistoryStore.fetchChangeHistoryList({
           parent: database.name,
+          view: ChangeHistoryView.CHANGE_HISTORY_VIEW_FULL,
```

Comment:
We probably don't want to view the full script for fetching the history list. We want to use basic view for getting the list (saving resources) and then use full view for getting the history item when needed.

Reply:
Updated